clients/fides-js/rollup.config.mjs

```diff
@@ -152,6 +152,9 @@ SCRIPTS.forEach(({ name, gzipErrorSizeKb, gzipWarnSizeKb, isExtension }) => {
         name: isExtension ? undefined : "Fides",
         format: isExtension ? undefined : "umd",
         sourcemap: IS_DEV ? "inline" : false,
+        amd: {
+          define: undefined, // prevent the bundle from registering itself as an AMD module, even if an AMD loader (like RequireJS) is present on the page. (FidesJS is not an AMD module.)
+        },
```

Comment:
Q: is the reason you can't use `format: "iife"` specifically because you still want the `cjs` output format to work? 

If so I think this is OK, but my comment would look like this:
```suggestion
        amd: {
          define: undefined, // prevent the bundle from registering itself as an AMD module, even if an AMD loader (like RequireJS) is present on the page. This allows FidesJS to use Rollup's `umd` format to support both `iife` and `cjs` modules, but excludes AMD.
        },
```

Reply:
I did try that and we still had issues that were getting really messy to try and solve. I had it mostly figured out in Admin UI and in Privacy Center, but then ALL Cypress tests were failing because things were missing on the window object. I got to that point after about 8 hours of playing around with it and opted to attempt this solution instead, which seems to be least risky and solves the edge case.